server.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+
+const app = express();
+const PORT = 3000;
```

Comment:
Часто удобно иметь возможность прокинуть порт снаружи:
```suggestion
const PORT = 3000;
const port = process.env.PORT || PORT;
```